ssh_key_link.go

```diff
@@ -0,0 +1,60 @@
+package scalr
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+)
+
+// Compile-time proof of interface implementation.
+var _ SSHKeysLinks = (*sshKeysLinks)(nil)
+
+type SSHKeysLinks interface {
+	Create(ctx context.Context, workspaceID string, sshKeyID string) (*SSHKeysLink, error)
+	Delete(ctx context.Context, workspaceID string) error
+}
+
+// sshKeysLinks implements SSHKeysLinks.
+type sshKeysLinks struct {
+	client *Client
+}
+
+// SSHKeysLink represents a single SSH key workspace link.
+type SSHKeysLink struct {
+	SSHKeyID string `jsonapi:"attr,ssh-key"`
+}
+
+type SSHKeysLinkCreateOptions struct {
+	SSHKeyID string `json:"ssh-key"`
+}
+
+// Create creates a SSH key workspace link.
+func (s *sshKeysLinks) Create(ctx context.Context, workspaceID string, sshKeyID string) (*SSHKeysLink, error) {
+	urlPath := fmt.Sprintf("workspaces/%s/ssh-key-links", url.QueryEscape(workspaceID))
+	linkOptions := SSHKeysLinkCreateOptions{
+		SSHKeyID: sshKeyID,
+	}
+	req, err := s.client.newJsonRequest("POST", urlPath, linkOptions)
```

Comment:
Something weird here, why do we create a json request, but later adding header for `vnd.api+json`?
```suggestion
	req, err := s.client.newRequest("POST", urlPath, linkOptions)
```